ci/scripts/quick-bazel.sh

```diff
@@ -8,7 +8,33 @@
 # This doesn't install dependencies, but should otherwise behave the
 # same as what CI would do on a pull request
 
-bazel query "rdeps(//..., //hw:verilator)" | \
-            sed -e 's/^/-/' | \
-            xargs bazel test --nobuild_tests_only --test_tag_filters=-cw310,-verilator -- //...
+set -e -x
+
+mkdir -p bazel-results
+
+# This queries for things that depend on //hw:verilator to exclude them
+bazel query \
+    'rdeps(//..., //hw:verilator)' \
+    | sed -e 's/^/-/' \
+    > bazel-results/slow_tests.txt
+
+run_tests() {
+    cat bazel-results/slow_tests.txt \
+        | xargs \
+            bazel test \
+            --keep_going \
+            --nobuild_tests_only \
+            --test_tag_filters=-broken,-cw310,-verilator \
+            -- //...
```

Comment:
This seems to be catching more targets than the PR description implies. Does there need to be a `-dv` in the tag filters?

Reply:
technically, the dv tests just echo the dvsim.py command to run to actually run the test, so it shouldn't hurt, but @a-will is right, we should probably just add a `-dv` flag to be explicit.